symforce/codegen/backends/cpp/cpp_config.py

```diff
@@ -32,6 +34,11 @@ class CppConfig(CodegenConfig):
         explicit_template_instantiation_types: Explicity instantiates templated functions in a `.cc`
             file for each given type. This allows the generated function to be compiled in its own
             translation unit. Useful for large functions which take a long time to compile.
+        override_methods: Add special function overrides in dictionary with symforce function keys
+            (e.g. sf.sin) and a string for the new method (e.g. fast_math::sin_lut), note that this bypasses
+            the default namespace (so std:: won't be added in front automatically)
+        extra_imports: Add extra imports to the file if you use custom overrides for some functions
+            (i.e. add fast_math.h)
```

Comment:
Include a note here that this only applies to functions created with `generate_function` - e.g. this doesn't let you define custom functions to use in the geo or cam type implementations (e.g. `rot3.h`)

Reply:
Yeah, good point, will do